item-spec/item-spec.md

```diff
@@ -21,18 +25,6 @@ required fields is a valid STAC Item.
   - Real world [implementations](../implementations.md) are also available.
 - [JSON Schema](json-schema/item.json)
 
-## WARNING
```

Comment:
Why has the warning and the Item fields header been removed?

Reply:
This is only in 3 of the many specs, and is also in the README.  I think it's time to take it out the specs, as a warning like this made more sense when the project was at a less mature stage.  I can re-add it you want.

Reply:
It's in all core JSON specs (item, collection, catalog). So if you remove it here, remove it in all others, too. But we should leave it open to @cholmes to decide when he thinks we are "stable" (1.0?).

Reply:
re-added it in the right place.